client/migrate.lua

```diff
@@ -0,0 +1,11 @@
+RegisterCommand("migratehouses", function()
```

Comment:
Is the idea that each player will need to run this command themselves to migrate their houses over?

Reply:
Can we just make this an admin only command that only needs to run once on setup and then you're done?

Reply:
No it only needs to be run once, however it needed to be on the client so it had a player it could callback to in order to obtain street and region data.

Reply:
If you had a better solution for this feel free to suggest.

Reply:
Ehh its good enough for the purpose it serves I suppose. If the command is run multiple times its not going to create dupes tho right?